newHometask/Person.java

```diff
@@ -0,0 +1,32 @@
+package newHometask;
+
+public class Person {
+    private String name;
+    private String surname;
+    private String sex;
+
+    private int age;
+
+    public Person(String name, String surname, String sex, int age) {
```

Comment:
Constructor with arguments is not a good option!!! (Let me describe why on the lesson)
Instead of this use no-args constructor + getters/setters